lockbox-ios/View/ItemDetailView.swift

```diff
@@ -212,10 +212,12 @@ extension ItemDetailView: UIGestureRecognizerDelegate {
 
 extension ItemDetailView: UITableViewDragDelegate {
     func tableView(_ tableView: UITableView, itemsForBeginning session: UIDragSession, at indexPath: IndexPath) -> [UIDragItem] {
-        let cell = tableView.cellForRow(at: indexPath)
-        let data = (cell as? ItemDetailCell)?.dragValue?.data(using: .utf8)
+        let cell = tableView.cellForRow(at: indexPath) as? ItemDetailCell
+        let data = cell?.dragValue?.data(using: .utf8)
         let itemProvider = NSItemProvider()
 
+        self.presenter?.dndStarted(itemId: self.itemId, value: cell?.titleLabel.text)
```

Comment:
Forgive my unfamiliarity ... this will be called when the drag starts.  I take it the completion closure below doesn't work for making other calls like this.

Is there any concern about over-counting because the drop doesn't happen?  I'm guessing it's a pretty rare occurrence, so that we're fine with the "noise".